app/assets/javascripts/discourse/tests/acceptance/fast-edit-test.js

```diff
@@ -73,4 +73,17 @@ acceptance("Fast Edit", function (needs) {
     assert.notOk(exists("#fast-edit-input"), "fast editor is not open");
     assert.ok(exists(".d-editor-input"), "the composer is open");
   });
+
+  test("Opens full composer when editing non-ascii characters", async function (assert) {
+    await visit("/t/internationalization-localization/280");
+
+    query("#post_2 .cooked").childNodes[0].innerHTML += `Don’t ”say doesn’t”`;
+    const textNode = query("#post_2 .cooked");
+
+    await selectText(textNode);
+    await click(".quote-button .quote-edit-label");
+
+    assert.notOk(exists("#fast-edit-input"), "fast editor is not open");
+    assert.ok(exists(".d-editor-input"), "the composer is open");
```

Comment:
Unrelated to the PR, but I recommend you have a look at https://github.com/mainmatter/qunit-dom/blob/master/API.md

This is a nicer way to express this kind of tests.